cumulus_etl/etl/studies/covid_symptom/covid_tasks.py

```diff
@@ -82,14 +83,14 @@ def add_error(self, docref: dict) -> None:
         with common.NdjsonWriter(error_path, "a") as writer:
             writer.write(docref)
 
-    async def read_entries(self) -> tasks.EntryIterator:
+    async def read_entries(self, *, progress: rich.progress.Progress = None) -> tasks.EntryIterator:
```

Comment:
is this * just futureproofing?

Reply:
Partly, but also to enforce current kwarg practices. For things like this that don't feel like the "noun" object of the "verb" of the method -- i.e. not `write_entries(rows)` but rather `write_entries(progress)` -- I prefer to forcibly relegate them to the kwargs portion with `*`. I think it helps readability, for myself anyway.

Reply:
gotcha - not my personal pref but a perfectly defensible thing